Mathlib/Tactic/Linter/DocPrime.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+
+/-!
+#  The "docPrime" linter
+
+The "docPrime" linter emits a warning on declarations that have no doc-string and whose
+name ends with a `'`.
+-/
+
+open Lean Elab
+
+namespace Mathlib.Linter
+
+/--
+The "docPrime" linter emits a warning on declarations that have no doc-string and whose
+name ends with a `'`.
```

Comment:
Can you extend this docstring slightly, using e.g.
- something like this: "They should have a doc-string, explaining (among others) why the primed name was chosen
(and how this declaration differs from its unprimed version, if applicable)."
- or part of your error message below (which I only saw later, and might be much nicer than this wording!)